luda-editor/luda-editor-client/src/app/sequence_list/events.rs

```diff
@@ -8,4 +8,11 @@ pub enum SequenceListEvent {
     SequenceLoadEvent {
```

Comment:
It would be better to name it like `SequenceLoadRequestedEvent` because when I read this event, I thought it's the event about loading, but it seems to make start loading sequence.

Personally I tried to make a event name just what exactly happened, not as what I have to do.
For example, I would name it like `SequenceTitleButtonClickEvent`, because user click the button. It's not their business to load sequence or not. I just let upper-side know that I click this button, then they will handle it, and they will give me next props, it just render about props.

Reply:
I think both of them would be better than now.
But I don't know which one is better between `SequenceLoadRequestedEvent` and `SequenceTitleButtonClickedEvent`.

I'll choose `SequenceTitleButtonClickedEvent` for the consistency of the code.

Reply:
Fixed with 7166c1e